includes/data-stores/class-wc-admin-reports-categories-data-store.php

```diff
@@ -165,12 +182,8 @@ protected function normalize_order_by( $order_by ) {
 	 * @return string
 	 */
 	protected function get_included_categories( $query_args ) {
```

Comment:
How about renaming to `get_included_categories_string` to differentiate from `get_included_categories_array`?

Reply:
Thanks for taking a look @psealock!

> How about renaming to `get_included_categories_string` to differentiate from `get_included_categories_array`?

I wondered the same, but I decided not to change it to keep it consistent with the methods defined in [`class-wc-admin-reports-data-store.php`](includes/data-stores/class-wc-admin-reports-data-store.php) (`get_included_products`, `get_included_variations`, `get_excluded_products`, `get_included_coupons`...). Otherwise, to keep them consistent, we should add `_string` at the end of the name of all of them, but I think that would just add noise to many other parts of the codebase.

What do you think?

Reply:
Sounds legit to me.